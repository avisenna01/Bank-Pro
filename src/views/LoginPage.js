import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Typography from "@material-ui/core/Typography";
import Grid from "@material-ui/core/Grid";



const useStyles = makeStyles(theme => ({
  root: {
    padding: theme.spacing(3, 9),
    backgroundColor: "#111113",
    borderRadius: "0px",
    height: "auto",

  },
  title: {
    fontSize: "70px",
    color: "white",
    fontFamily: 'SF Pro Text,SF Pro Icons,Helvetica Neue,Helvetica,Arial,sans-serif',
    textAlign: "center",
    marginTop: "160px"
  },
}));

const inputBox = {

  border: "4px solid #66BBFF",
  marginTop: "15px",
  width: "250px",
  backgroundColor: "white",
  height: "40px",
  borderRadius: "10px",
  fontSize: "20px",
  padding: "5px",
  paddingLeft: "15px"
}

const LoginPage = props => {

  const classes = useStyles();

  const handleChange = event => {
    console.log(event.target.value)
  };


  return (
    <Grid container className={classes.root}>
      <Grid item xs={12}>
        <Typography className={classes.title}>Bank Pro</Typography>
      </Grid>
      <Grid item xs={12} container justify="center" alignItems="center" style={{ paddingBottom: "400px" }}>
        <input onChange={handleChange} style={inputBox} type="text" placeholder="Account Number" />
      </Grid>
    </Grid>
  );
};

export default LoginPage;
