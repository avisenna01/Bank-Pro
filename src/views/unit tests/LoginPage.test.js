import React from 'react'
import LoginPage from "../LoginPage";
import renderer from 'react-test-renderer';
import ReactDOM from "react-dom";
import { shallow } from 'enzyme';

const rendercomponent = () => (shallow(<LoginPage />));

describe('LoginPage page try 1', () => {
  it('snapshot renders', () => {
    const renderedcomponent = rendercomponent();
    expect(renderedcomponent.find('div').length).toEqual(0);
  });
});

describe('LoginPage page try 2', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<LoginPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<LoginPage />, div);
});