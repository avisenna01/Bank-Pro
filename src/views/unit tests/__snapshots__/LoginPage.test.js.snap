// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage page try 2 snapshot renders 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <p
      className="MuiTypography-root makeStyles-title-2 MuiTypography-body1"
    >
      Bank Pro
    </p>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
    style={
      Object {
        "paddingBottom": "400px",
      }
    }
  >
    <input
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Account Number"
      style={
        Object {
          "backgroundColor": "white",
          "border": "4px solid #66BBFF",
          "borderRadius": "10px",
          "fontSize": "20px",
          "height": "40px",
          "marginTop": "15px",
          "padding": "5px",
          "paddingLeft": "15px",
          "width": "250px",
        }
      }
      type="text"
    />
  </div>
</div>
`;
